mobile-widgets/qml/DownloadFromDiveComputer.qml

```diff
@@ -169,6 +169,7 @@ Kirigami.Page {
 						downloadThread.deviceData.bluetoothMode = true
 					else
 						downloadThread.deviceData.bluetoothMode = false
+                    downloadThread.deviceData.devName = comboConnection.currentText
```

Comment:
that does look like you still have space vs. tab problems in your editor settings.